src/targets/gpu/fuse_ops.cpp

```diff
@@ -184,30 +184,32 @@ MIGRAPHX_PRED_MATCHER(fusable_conv, instruction_ref ins)
         return false;
     if(enabled(MIGRAPHX_DISABLE_MIOPEN_FUSION{}))
         return false;
-    if(ins->name() != "gpu::convolution")
+    if(ins->name() != "gpu::miopen_convolution")
+        return false;
+    auto miopen_conv_op = any_cast<miopen_convolution>(ins->get_operator());
```

Comment:
Use `ins->get_operator().to_value()`, and remove the include for `migraphx/gpu/convolution.hpp`.

Reply:
Two things are preventing me from doing so:
1. `miopen_conv_bias` construction requries `op::convolution`.  I need to cast `miopen_convolution<op::convolution>`. to get that `op::convolution` object.  I can make it work if i can some how reflect "op" object from `miopen_convolution`. which is not working with templated `miopen_convolution`. 

2. `fuse_ops` pass refers to `algo` (e.g  miopenConvolutionFwdWinograd) in many places. which is not reflected currently.

Reply:
> miopen_conv_bias construction requries op::convolution. 

You can deserialize the operator:

```cpp
auto v = ins->get_operator().to_value();
auto conv_op = from_value<op::convolution>(v["op"]);
```

> fuse_ops pass refers to algo (e.g miopenConvolutionFwdWinograd) in many places. which is not reflected currently.

Lets add that to the reflect method then.